docs/docs/policies.mdx

```diff
@@ -396,6 +398,308 @@ for details.
+|                                       |                        | intents. You might want to do this if you come across a      |
+|                                       |                        | certain list of intents for which there are too many false   |
+|                                       |                        | warnings generated.                                          |
++---------------------------------------+------------------------+--------------------------------------------------------------+
+| tolerance                             | 0.0                    | The `tolerance` parameter is a number that ranges from `0.0` |
+|                                       |                        | to `1.0` (inclusive). It helps to adjust the threshold score |
+|                                       |                        | used during prediction of `action_unlikely_intent` at        |
+|                                       |                        | inference time. Here, `0.0` means that the score threshold   |
+|                                       |                        | is the one that `IntentTEDPolicy` had determined at training |
+|                                       |                        | time. A tolerance of `1.0` means that the threshold score    |
+|                                       |                        | is so low that `IntentTED` would not trigger                 |
+|                                       |                        | `action_unlikely_intent` for any of the "negative examples"  |
+|                                       |                        | that it has encountered during training. These negative      |
+|                                       |                        | examples are combinations of dialogue contexts and user      |
+|                                       |                        | intents that are _incorrect_. `IntentTEDPolicy` generates    |
+|                                       |                        | these negative examples from your training data by picking a |
+|                                       |                        | random story part and pairing it with a random intent that   |
+|                                       |                        | doesn't occur at this point.                                 |
++---------------------------------------+------------------------+--------------------------------------------------------------+
+```
+
+</details>
+
+#### Tuning the tolerance parameter
+
+When [reviewing real conversations](./conversation-driven-development.mdx#review), users are
+ encouraged to tune the `tolerance` parameter in `IntentTEDPolicy`'s configuration to reduce the number
+of false warnings (intents that actually are likely given the conversation context).
+As you increase the value of `tolerance` from `0` to `1` in steps of `0.05`,
```

Comment:
Why `0.05`?

Reply:
We have that in the code. We didn't make it flexible / customizable because we don't see a need for it to be as fine-grained as `0.01` right now based on our experiments.